Framework/DataHandling/test/LoadPLNTest.h

```diff
@@ -0,0 +1,85 @@
+#ifndef LOADPLNTEST_H_
+#define LOADPLNTEST_H_
+
+#include <cxxtest/TestSuite.h>
+
+#include "MantidAPI/AnalysisDataService.h"
+#include "MantidAPI/FrameworkManager.h"
+#include "MantidAPI/Run.h"
+#include "MantidAPI/WorkspaceFactory.h"
```

Comment:
Avoid the `WorkspaceFactory` and prefer `WorkspaceCreation`. But can it be that this include is not used?

Reply:
I removed the include files that were not used - it included WorkspaceFactory.

Reply:
@reimundILL Hi, are there any other issues I need to address?

Reply:
@geishm-ansto I'll have a closer look